README.md

```diff
@@ -199,6 +199,7 @@ jobs:
         with:
           token: ${{ secrets.GITHUB_TOKEN }}
           config: .github/reviewers.yml # Config file location override
+          use_local: true # Look for config locally during run instead of in repo
```

Comment:
As it's not very easy to understand how you can use this, I'd like to explain this bit further. What about

```suggestion
          # Look for config locally during run instead of in repo.
          # For instance, if you'd like to use a config file stored in external storage,
          # you can fetch it before you run this action, then let this action pick it with `use_local` `true`.
          # See https://github.com/necojackarc/auto-request-review/issues/76 for more details.
          use_local: true
```

This is my understanding. Correct me if I'm wrong, or feel free to improve the wording 👍

Reply:
Looks good to me, updated